src/HW_models/core_cm33.h

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
Maybe we should extract out a common file? We can make `core_cmxx.h` and let `core_cm33.h` and `core_cm4.h` include this file?